### ISSUE TITLE
The rule description for MA0171 is incorrect

### ISSUE THREAD

**Issue by alexonthespot7** [meziantou/Meziantou.Analyzer#891]
### Version of the Meziantou.Analyzer NuGet package

2.0.227

### Rule Identifier

MA0171

### Target Framework

any

### C# Language version

any

### Description

The rule description for MA0171 "Use pattern matching instead of inequality operators for discrete value" is incorrect and just copy-pastes the description of rule MA0149. The description of rule MA0171 should be corrected to reflect the actual semantics of the rule.

### Reproduction Steps

Minimal code:

```c#
code to reproduce the error
```

**Comment by meziantou** [comment]
Thanks for reporting it!

Fix by 3d7df2e31637295e94ce664284e75a2378725e95
